Test failure in IfritTests.Fuse_TestsTokenize/testProtocolWeightedSearchTokenized

```
% swift test --filter IfritTests.Fuse_TestsTokenize/testProtocolWeightedSearchTokenized
```

```
/Users/dfabulich/git/Ifrit/Tests/Ifrit_FuseTests/Fuse_TestsTokenize.swift:74: error: -[IfritTests.Fuse_TestsTokenize testProtocolWeightedSearchTokenized] : XCTAssertTrue failed - The first result is the first book
/Users/dfabulich/git/Ifrit/Tests/Ifrit_FuseTests/Fuse_TestsTokenize.swift:75: error: -[IfritTests.Fuse_TestsTokenize testProtocolWeightedSearchTokenized] : XCTAssertTrue failed - The second result is the second book
```

I ran `git bisect` and found that it started failing in 6cde912baae9ad871b2173bc01e6e9479212c9ec

--- a/Sources/Ifrit/Classes/Fuse.swift
+++ b/Sources/Ifrit/Classes/Fuse.swift
@@ -22,7 +22,7 @@ public class Fuse: @unchecked Sendable {
     ///   - tokenize: When true, the search algorithm will search individual words **and** the full string, computing the final score as a function of both. Note that when `tokenize` is `true`, the `threshold`, `distance`, and `location` are inconsequential for individual tokens.
     ///    `objSortStrategy:` if you search in object by property with array - you can choose your sorting strategy.  By default is `.minimalScore`.
     ///   - qos: quality-of-service, use this to set search task priority. Better never use `.user-interactive`. By default is `.userInitiated`.
-    public init (location: Int = 0, distance: Int = 100, threshold: Double = 0.6, isCaseSensitive: Bool = false, tokenize: Bool = false, objSortStrategy: ObjectsSortStrategy = .minimalScore, qos: DispatchQoS = .userInitiated) {
+    public init (location: Int = 0, distance: Int = 100, threshold: Double = 0.6, isCaseSensitive: Bool = false, tokenize: Bool = false, objSortStrategy: ObjectsSortStrategy = .averageScore, qos: DispatchQoS = .userInitiated) {
         self.location = location
         self.distance = distance
         self.threshold = threshold

